addons/project/models/project_task.py

```diff
@@ -349,6 +349,13 @@ def _compute_display_in_project(self):
                 not record.parent_id or record.project_id != record.parent_id.project_id
             )
 
+    def _inverse_parent_id(self):
+        for task in self:
+            if not task.parent_id:
+                self.display_in_project = True
```

Comment:
is it task or self ?

Reply:
if it's task, i think there is no need to write 2 times, you can create a variable display_in_project, then write one time at the end

Reply:
ah good catch, indeed it is task. 🙈